egs/librispeech/WSASR/conformer_ctc2/train.py

```diff
@@ -31,6 +31,7 @@
   --exp-dir conformer_ctc2/exp \
   --lang-dir data/lang_bpe_200 \
   --otc-token "<star>" \
+  --num-features 768 \
```

Comment:
```suggestion
  --feature-dim 768 \
```

or just delete it since the default value is 768